src/app/shared/google-analytics/service/analytics.service.ts

```diff
@@ -70,4 +70,12 @@ export class AnalyticsService {
   public rateComplexNavigator(rating: number) {
     this.analytics.event(Action[Action.RatingComplexNavigator], Category[Category.complexNavigator], 'rating', rating);
   }
+
+  public useOrthologyButton(action: Action) {
```

Comment:
Rename this method, as it is more generic than ortholog button. It can be `useComplexNavigatorSettingButton`.